### PR TITLE
PHPLIB-624 Sync change stream spec tests for delta oplog entries

### PR DESCRIPTION
**Pull request by alcaeus**
PHPLIB-624

### CODE REVIEW

File: tests/UnifiedSpecTests/UnifiedSpecTest.php
Comment (jmikola) [inline tests/UnifiedSpecTests/UnifiedSpecTest.php:192]: Same comment as https://github.com/mongodb/mongo-php-library/pull/811#discussion_r598996504. I'll note that this PR is actually adding a file to `change-stream/`, so perhaps this is needed to actually run the test. If so, this can stay.

**Review by jmikola — APPROVED** [review]

File: tests/UnifiedSpecTests/UnifiedSpecTest.php
Comment (alcaeus) [reply tests/UnifiedSpecTests/UnifiedSpecTest.php:192]: Yes, same story here. Happy to hold off on this until we've come to a resolution or until the changes introduced by #806 are merged.

Comment (jmikola) [reply tests/UnifiedSpecTests/UnifiedSpecTest.php:192]: Oh, they can both get merged so these tests can be executed. In #811 I missed that you were adding a unified CRUD test.

**Review by jmikola — APPROVED** [review]

**Pull request merged**
